pkg/scheduler/queue/user_queues_test.go

```diff
@@ -457,6 +458,36 @@ func TestGetOrAddQueueShouldUpdateProperties(t *testing.T) {
 	}
 }
 
+func TestGetOrAddQueueConcurrency(t *testing.T) {
```

Comment:
Is the test missing some assertions? Or was it just for testing for -race conditions?

Reply:
Yup, it's just to test with `-race` condition, which is part of the repo ci